app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/ui/theme/Theme.kt

```diff
@@ -0,0 +1,38 @@
+package au.com.gridstone.trainingkotlin.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+
+private val DarkColorPalette = darkColors(
+  primary = Primary,
+  primaryVariant = PrimaryDark,
+  secondary = Secondary
+)
+
+private val LightColorPalette = lightColors(
+  primary = Primary,
+  primaryVariant = PrimaryDark,
+  secondary = Secondary
+)
```

Comment:
explicit type, according to [the style guide](https://github.com/Gridstone/KotlinStyleGuide#type-inference)